src/DynamoCore/Graph/Workspaces/WorkspaceInfo.cs

```diff
@@ -15,6 +15,7 @@ namespace Dynamo.Graph.Workspaces
     /// </summary>
     public class WorkspaceInfo
     {
+        private const string dynamo1HomeWorkspaceNameString = "Home";
```

Comment:
so irrespective of languages, "Home" is always a "Home" - can you write down a small comment here? so we don't repeat the same for 3.0?

Reply:
sure will do.
in I called this Dynamo1Home... as in Dynamo 2 the name of homeWorkspaces can now to be set to other values, like the filePath... but it's not an issue as in json we have the customNode flag.